Weather/Weather/Scenes/Start/StartScreenViewController.swift

```diff
@@ -8,31 +8,72 @@
 
 import UIKit
 import CoreLocation
-import Then
 
 final class StartScreenViewController: UIViewController {
+    @IBOutlet private weak var locationView: UIView!
+    @IBOutlet private weak var activityIndicator: UIActivityIndicatorView!
+    
+    private let startRepos = StartRepository()
+    private let locationManager = CLLocationManager().then {
+        $0.desiredAccuracy = kCLLocationAccuracyKilometer
+        $0.distanceFilter = 5  // In kilometer.
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        locationManager.delegate = self
     }
     
-    @IBAction private func locationServicesEnable(_ sender: UIButton) {
-        // allow access location services
+    private func configLocationView(_ isHidden: Bool) {
+        locationView.isHidden = !isHidden
+        activityIndicator.isHidden = isHidden
+    }
+    
+    private func updateData(with location: Location?) {
+        configLocationView(false)
+        guard let location = location else {
```

Comment:
```suggestion
        guard let location = location else { return }
```
nếu câu lệnh ngắn thì có thể viết liền 1 dòng như thế này cũng đc